tests/create_account.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('Create Account Flow - Step 3 & Step 4', () => {
+
+    test.beforeEach(async ({ page }) => {
+        await page.goto('http://127.0.0.1:8080/create');
```

Comment:
```suggestion
        await page.goto('http://127.0.0.1:8080/create');
```
now tagert https://voice-korea.dev.biyard.co

Reply:
and after all `goto` , make screenshots to be confirmed by manual.

Reply:
@dotandev target must be https://voice-korea.dev.biyard.co/en